src/components/BottomBar/SkyBrowser/SkyBrowserNearestImagesList.jsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { useSelector, shallowEqual } from 'react-redux';
+import PropTypes from 'prop-types';
+import CenteredLabel from '../../common/CenteredLabel/CenteredLabel';
+import {FilterList, FilterListData} from '../../common/FilterList/FilterList';
+import styles from './SkyBrowserNearestImagesList.scss';
+import SkyBrowserFocusEntry from './SkyBrowserFocusEntry';
+
+function SkyBrowserNearestImagesList({
+  activeImage,
+  height,
+  selectImage,
+  currentBrowserColor
+}) {
+  const fov = useSelector((state) => {
+    return state.skybrowser.browsers[state.skybrowser.selectedBrowserId].fov;
+  }, shallowEqual);
+  const ra = useSelector((state) => {
+    return state.skybrowser.browsers[state.skybrowser.selectedBrowserId].ra;
+  }, shallowEqual);
+  const dec = useSelector((state) => {
+    return state.skybrowser.browsers[state.skybrowser.selectedBrowserId].dec;
+  }, shallowEqual);
+  const cartesianDirection = useSelector((state) => {
+    return state.skybrowser.browsers[state.skybrowser.selectedBrowserId].cartesianDirection;
+  }, shallowEqual);
+  const luaApi = useSelector((state) => {
+    return state.luaApi
+  }, shallowEqual);
+  const imageList = useSelector((state) => {
+    return state.skybrowser.imageList
+  }, shallowEqual);
+
+  const distanceSortThreshold = 0.1;
+
+  function euclidianDistance(a, b) {
```

Comment:
This is where I really wished we used typescript... hard to understand what a and b is here. Coordinate vectors?

Could be fixed by using more meningful names perhaps?